tests/documentation_tests/scripts/update-docs-examples.sh

```diff
@@ -28,14 +28,15 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ROOT_DIR="$(cd "$SCRIPT_DIR/../../.." && pwd)"
 # Set the documentation directory to the docs/ directory
 DOCS_DIR="$ROOT_DIR/docs"
-# Set the test directory to the test/documentation_tests/ directory
-TEST_DIR="$ROOT_DIR/tests/documentation_tests"
-# Set the output directory to the test/documentation_tests/ directory
-OUTPUT_DIR="$TEST_DIR"
 # Set the files to exclude from the copy
 EXCLUDE_FILES=(
   "sailoperator.io.md" "guidelines.md"
 )
+# Directory to store the output files
+TEST_DIR="${TEST_DIR:-$(mktemp -d)}"
```

Comment:
```suggestion
ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
TEST_DIR="${ARTIFACTS}/docs.test"
```

Shouldn't it default to the same values as in the run script?

Also since it's not really expected to run this script directly, you can alternatively just check that the variable is set which makes more sense IMHO.